research/integrations/fedora-ci/index.md

```diff
@@ -0,0 +1,338 @@
+
+Support for side tags has been requested in the original thread with the
+proposal. This may be required for “packaging workflows” that are more complex,
+e.g., Rust packaging (dependencies that are built as separate packages).
+
+> As a maintainer, I should be able to specify the dependencies of my package
+> that need to be verified with any proposed update.
+
+### Upstream vs Downstream
+
+By default, we sync the Packit config to the downstream. This can cause a
+conflict when Packit runs as a Fedora CI, since the config would override any
+defaults that we would've set.
+
+Additionally, there needs to be a way to distinguish between upstream/downstream
+jobs anyways, cause Testing Farm runs on the downstream install RPMs from the
+Koji rather than Copr.
+
+:::warning
+
+It would probably be easier to run stateless as the current CI, utilizing only
+defaults and ignoring any Packit config that is present.
+
+:::
+
+## User perspective
+
+### Fedora
+
+> As a maintainer I'd like to open a PR and have automated RPM build to verify
+> that, whatever I'm submitting, can be
+>
+> - built,
+> - installed, and
+> - passes the test suite.
+>
+> Adding and configuring the tests in the dist-git should be my responsibility.
+> I don't want the CI to be blocking, just annoying…
+
+**TODO**:
+
+- [ ] Packit should be able to run without any repo-specific configuration, but
+      there should be a way to override, if needed.
+
+### Fedora SIGs
+
+AFAIK the repositories are hosted on the Fedora dist-git, therefore they should
+be handled by the Fedora CI already, the experience should be similar (if not
+the same) to the regular Fedora packages.
+
+### CentOS SIGs
+
+As there is no streamlined CI for the CentOS SIGs, adoption would allow
+relatively simple integration with the Testing Farm.
+
+## Difficulties depending on the forge
+
+### Pagure
+
+Mainly reliability, it would require retries. Together with normal Packit CI
+workflows, considerable overhead.
+
+### GitLab
+
+ogr has a support for GitLab already, reporting sucks a bit, but it's not
+a blocking issue (due to the lack of upstream users, the priority of improving
+the UX has been lowered).
+
+#### Upstream vs downstream
+
+Difference between upstream and downstream events can be told easily by source
+of the webhooks (downstream come from Fedora Messaging, upstream directly from
+GitLab instances).
+
+### Forgejo
+
+Forgejo is an open-source git forge that seems to be a viable alternative to
+both Pagure and GitLab. There is no support for the forgejo in ogr as of now,
+additionally to ogr support we would need to adjust parsing of the webhooks, but
+it can completely replace the Pagure (as Fedora dist-git is the only Pagure
+instance used by Packit).
+
+## Notes from the arch discussion
+
+### Configuration
+
+For the beginning we should just handle the packages that are configured.
+
+:::warning
+
+This means that we don't want to handle _all_ Fedora packages for the _PoC/MVP_.
+
+:::
```

Comment:
Yes, but count on this possibility  (or global config ala zuul) since this should be the ultimate state.